tools/typescript2fabel.js

```diff
@@ -266,6 +273,12 @@ function visitModule(node) {
             case ts.SyntaxKind.FunctionDeclaration:
                 mod.methods.push(getMethod(node, true));
                 break;
+            case ts.SyntaxKind.ModuleDeclaration:
+                modules.push(visitModule(node, modules));
+                break;
```

Comment:
I wonder why you make the nested modules siblings of their parent? Does this work that way? Can you point a sample of the kind of modules this is trying to deal with?